purchases/src/main/kotlin/com/revenuecat/purchases/paywalls/components/properties/VideoUrls.kt

```diff
@@ -0,0 +1,36 @@
+package com.revenuecat.purchases.paywalls.components.properties
+
+import androidx.compose.runtime.Immutable
+import com.revenuecat.purchases.InternalRevenueCatAPI
+import com.revenuecat.purchases.utils.serializers.URLSerializer
+import dev.drewhamilton.poko.Poko
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+import java.net.URL
+
+@InternalRevenueCatAPI
+@Poko
+@Serializable
+@Immutable
+class ThemeVideoUrls(
+    @get:JvmSynthetic val light: VideoUrls,
+    @get:JvmSynthetic val dark: VideoUrls?,
+)
+
+@InternalRevenueCatAPI
+@Poko
+@Serializable
+@Immutable
+class VideoUrls(
+    @get:JvmSynthetic
+    val width: UInt,
+    @get:JvmSynthetic
+    val height: UInt,
+    @get:JvmSynthetic
+    @Serializable(with = URLSerializer::class)
+    val url: URL,
```

Comment:
I wonder... should we allow developers to set a reference to an embedded video? So basically, follow a similar pattern as we do with fonts. If we find the name of the font in the resources for the app, we use that video. If not, we download it. That might with bandwidth... But then again, it's probably a future improvement we can do without causing a breaking change, so I think this is ok for now.